README.md

```diff
@@ -452,12 +453,20 @@ By default `KNAPSACK_PRO_REPOSITORY_ADAPTER` variable has no value so knapsack_p
 
 You can also use git as repository adapter to determine branch and commit hash, please see below section.
 
-#### When should you set global variable `KNAPSACK_PRO_REPOSITORY_ADAPTER=git` (required when CI provider is not supported)
+#### When should you set global variable `KNAPSACK_PRO_REPOSITORY_ADAPTER=git` (when CI provider is not supported and you use git)
 
 `KNAPSACK_PRO_REPOSITORY_ADAPTER` - When it has the value `git`, your local version of git on CI server will be used to get the branch name and commit hash. You also need to set `KNAPSACK_PRO_PROJECT_DIR` with the project directory path.
 
 `KNAPSACK_PRO_PROJECT_DIR` - Path to the project on the CI node, for instance `/home/ubuntu/my-app-repository`. It should be the top-level directory of your repository.
 
+#### When you don't use git
+
+If your CI provider does not expose commit hash and branch name as environment variable then `knapsack_pro` gem does not know what is commit hash and branch name.
+You can manually set below values as environment variables:
```

Comment:
It occurred to me that we are really not listing _the values_ one should set here. Just the variables in which the actual values should be stored. So maybe something like that would work better?:

`You can manually set the values of the current commit hash and branch name in the environment variables:`

Or maybe `You can manually set these values...`.